src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -176,17 +177,60 @@ void should_fail_correctly_if_document_is_duplicate_or_document_is_already_attac
             .eventId(EVENT_ID_ATTACH_TO_CASE)
             .build();
 
+        ValidatableResponse response = given()
+            .body(callbackRequest)
+            .headers(userHeaders())
+            .post(CALLBACK_ATTACH_CASE_PATH)
+            .then()
+            .statusCode(200);
+
+        verifySuccessResponse(response, callbackRequest);
+        verify(exactly(1), startEventRequest());
+        verify(exactly(1), submittedScannedRecords());
+    }
+
+    @DisplayName("Should attach missing document and skip already attached one")
+    @Test
+    void should_fail_when_duplicate_found_with_mismatching_exception_record_reference() throws JsonProcessingException {
```

Comment:
name and display name are different

Reply:
ah. forgot to edit this one. i'll just delete this display name